boot/freeldr/freeldr/include/ntldr/winldr.h

```diff
@@ -17,6 +17,31 @@
 // Some definitions
 
 #include <pshpack1.h>
+typedef struct /* ACPI Standard Header */
```

Comment:
Would it be possible to reuse some existing definitions, rather than duplicating them?
Maybe from
https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/include/reactos/drivers/acpi/acpi.h;hb=HEAD

Reply:
The code is from ReactOS and not Freeldr.